config/policies.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  authRevalidationMinutes: 30,
+};
```

Comment:
added this new config file as a place to put policy-related config, such as this `authRevalidationMinutes` value